packages/content/src/MediaGallery/index.js

```diff
@@ -230,6 +229,7 @@ class MediaGallery extends Component {
                   <ActiveThumbnailButton
                     type="button"
                     aria-controls="hwGallery_slide"
+                    className="thumbnailButton"
```

Comment:
Is this a namespaced/safe enough class name?

Reply:
not being used ... going to take it out next version (which is incoming!)